src/consensus/vote.hpp

```diff
@@ -251,4 +191,82 @@ class NextVotesForPreviousRound {
   LOG_OBJECTS_DEFINE
 };
 
+struct votesBundle {
+  bool enough;
+  blk_hash_t voted_block_hash;
+  std::vector<Vote> votes;  // exactly 2t+1 votes
+
+  votesBundle() : enough(false), voted_block_hash(blk_hash_t(0)) {}
+  votesBundle(bool const enough_, blk_hash_t const& voted_block_hash_, std::vector<Vote> const& votes_)
```

Comment:
- uff - bool const ?  
- you just exchanged formatting here, "enough_" with "_" should be name of class member, not ctor parameter...